recipes-openxt/qemu-dm/qemu-dm-2.6.2/block-remove-unused-block-format-support.patch

```diff
@@ -0,0 +1,79 @@
+From 5e9813ea9c6794307937af2c8bf709b99e065468 Mon Sep 17 00:00:00 2001
+From: Jason Andryuk <jandryuk@gmail.com>
+Date: Wed, 17 May 2017 14:46:46 -0400
+Subject: [PATCH 1/2] block: remove unused block format support
+
+block.c need to be modified to remove references to the disabled qcow2
+format.
+---
+ block.c             |  6 ++++--
+ block/Makefile.objs | 15 ++++++++-------
+ 2 files changed, 12 insertions(+), 9 deletions(-)
+
+diff --git a/block.c b/block.c
+index 951f5f4..3bad8ef 100644
+--- a/block.c
++++ b/block.c
+@@ -1405,7 +1405,7 @@ done:
+     qdict_del(options, bdref_key);
+     return c;
+ }
+-
++#if 0
```

Comment:
The patch description was supposed to explain this: "block.c need to be modified to remove references to the disabled qcow2 format."  I can add a comment at the ifdef location.

We don't use qemu for disk snapshots, right?